ohttp.go

```diff
@@ -418,6 +418,30 @@ func (g Gateway) Config(keyID uint8) (PublicConfig, error) {
 	return PublicConfig{}, fmt.Errorf("unknown keyID %d", keyID)
 }
 
+func (g *Gateway) DropConfig(keyID uint8) error {
+	if _, ok := g.keyMap[keyID]; ok {
+		delete(g.keyMap, keyID)
+		newKeys := make([]uint8, 0)
```

Comment:
can be as follow and access the array by indices
```suggestion
		newKeys := make([]uint8, len(g.keys) - 1)
```

Reply:
I believe it should be ```make([]uint8, 0, len(g.keys) - 1)``` so the init size is 0 and the capacity is correct.